src/components/molecules/TextOverlay.jsx

```diff
@@ -139,10 +124,11 @@ const StyledTextContainer = styled(({ isSidePreview, ...other }) => <SimpleBar {
   margin-top: 0.5rem;
   margin-bottom: 0;
   width: 100%;
-  padding-right: 15px;
+  /* padding-right: 15px; */
   max-height: ${({ isSidePreview }) => (isSidePreview ? '150px' : '300px')};
   overflow-y: auto;
   overflow-x: hidden;
+  text-align: center;
```

Comment:
Text box has editor which allows users to change text alignment
Won't this overwrite users settings?

Reply:
This element is used on Text Icon which is shown on a modal popup ...

Reply:
That's right, sorry